### ISSUE TITLE
Add utility to split datasets in train + test set

### ISSUE THREAD

**Issue by targos** [mljs/cross-validation#6]
Prior art: http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html#sklearn.model_selection.train_test_split
